exercises/diffie-hellman/example.py

```diff
@@ -0,0 +1,13 @@
+import random
+
+
+def private_key(p):
+    return random.randint(2, p-1)
```

Comment:
I think that this should use `secrets` rather than `random`, since the exercise is a cryptographic one. Perhaps we should also have a note in README.md that solutions should avoid using `random` since it's not cryptographically secure?

```
def private_key(p):
    return 2 + secrets.randbelow(p-2)
```

Reply:
I did look at that, but according to the [PEP](https://www.python.org/dev/peps/pep-0506/) it was introduced in version 3.6, so it isn't compatible with Python 2 or most Python 3 versions on stable distros. It seems that exercism wants cross compatibility which is why I chose to use the default random instead.

Reply:
Good points!

These are just training exercises, so `random` is good enough I think.
But it would be a really good idea to mention `secrets` in `HINTS.md` for such exercises.

@N-Parsons, could you please create an issue for it to not forget?

Reply:
I'll add a note to the readme about it and pseudo randomness.

Reply:
Sorry, @kusti8, I hadn't realised that `secrets` was only in Python 3.6.

@m-a-ge, I'll create an issue for it now :)